### DISCUSSION TITLE
Map works only on Windows servers, but not on Linux servers.

### DISCUSSION THREAD

**Discussion by CarlosPedretti** [SmartlyDressedGames/Unturned-3.x-Community#4339] [Q&A]
I have a very nostalgic map for me that I dedicated a lot of time to. Its creation was in 2020 and currently I modified it a little. It turns out that the initial map, the one from 2020, has the same error as its due update.

That is, when the map is established on a server with a Linux operating system, the map gives you the "Modified version of the map" error. 

However, on Windows it works perfectly! It must be an incompatibility with Linux, but I don't know which file is causing this problem. 

Please I need help. I refuse to abandon this map that is so dear to me. 
In advance, thank you very much for your attention.

This is the link to the map in case someone wants to review the files to corroborate any irregularities.

https://steamcommunity.com/sharedfiles/filedetails/?id=3154968840

**Comment by DrDallas1** [comment]
Can you post your Client.log? (For joining on Linux)

**Reply by CarlosPedretti** [reply]
Yes, give me a second

**Comment by CarlosPedretti** [comment]
Here are the logs.
[Server Log.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/14213388/Server.Log.log)
[Server Prev Log.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/14213389/Server.Prev.Log.log)
[Client Log.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/14213390/Client.Log.log)

**Reply by SDGNelson** [reply]
It appears there are a lot of missing assets on both the client and server. In trying to repair them perhaps it's falling back to different assets on Linux than Windows? Best option would be to determine which mod(s) might be missing that contain those assets.

Could also be this one: https://support.smartlydressedgames.com/hc/en-us/articles/12295511417364-Why-does-my-mod-work-on-Windows-but-not-Linux

**Reply by CarlosPedretti** [reply]
The reality is that I don't remember the dependencies of the map well, of course I have the main ones which are the Assets of Hawaii and Greece. However, there may be others that i am not aware of.
Is there any way to delete these objects without knowing the presidency of the mod?

By the way, your resolution seems very logical to me, and it would give meaning to all the visible characteristics regarding the error.

I had previously read the article that you passed, but I did not find any error regarding that on the map. At least with my basic knowledge of Linux I was not able to see anything out of the ordinary.

**Reply by GazziFX** [reply]
Yes, you can add `-NoPreserveMissingObjects` command line argument to the game and then load and save level

**Reply by CarlosPedretti** [reply]
Okay, thank you very much!

Tomorrow I will be doing tests, my tests today were total failures. Until now I haven't found the solution.

**Reply by CarlosPedretti** [reply]
Hello, I just tried adding that line in the launcher and it doesn't work that way either. I have checked all the files, their definitions are correct and they are exactly the same, both in names and sizes. (Server and Client files)
I honestly don't know what to do and I have no idea what causes it.
